LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2015 President and Fellows of Harvard College
+Copyright (C) 2016 Python LTI Initiative
```

Comment:
OK, so my knowledge in this area is pretty shallow. Since the license includes the line `The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.` it seems like we need to preserve the chain of earlier copyright statements + add our own. Also it looks like I screwed up when I removed the Tophat Monacle copyright statement when I forked. I'm basing this on http://programmers.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project